src/main/java/core/basesyntax/service/impl/AuthorizationServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.exception.WrongLoginOrPasswordException;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.User;
+import core.basesyntax.service.AuthorizationService;
+
+public class AuthorizationServiceImpl implements AuthorizationService {
+    @Inject
+    private UserDao userDao;
+
+    public User authorize(String login, String password) throws WrongLoginOrPasswordException {
+        if (login == null || password == null) {
+            throw new RuntimeException("Expected to receive data, but got null");
```

Comment:
```suggestion
            throw new IllegalArgumentException("Expected to receive data, but got null");
```

same everywhere where you expect to receive data and not null